docs/organizations/managing-security-and-risk.md

```diff
@@ -105,16 +158,57 @@ The following table describes how item statuses map to deadlines:
+| Low                        | 120                | Minor                              | Low and other                    |
+
+## Managing access to security and risk management
+
+To better track and address security vulnerabilities, organization admins can extend access to security and risk management by assigning the Security Manager role to organization members.
+
+The Security Manager role is independent of the Git provider role of an organization member and provides only the permissions necessary to monitor security vulnerabilities, following the principle of least privilege:
+
+| Permission                                                  | Organization Admin | Security Manager           |
+|-------------------------------------------------------------|--------------------|----------------------------|
+| Access security and risk management                         | Yes                | Yes                        |
+| Access associated issues (Codacy repositories, Jira issues) | Yes                | Keeps original permissions |
+| Manage integrations                                         | Yes                | Yes                        |
+| Assign and revoke the Security Manager role                 | Yes                | No                         |
+| All other Organization Admin permissions                    | Yes                | No                         |
+
+To assign the Security Manager role:
 
-| Security issue severity | Item priority | Days to close |
-|-------------------------|---------------|---------------|
-| Critical                | Critical      | 30            |
-| Medium                  | Medium        | 90            |
-| Minor                   | Low           | 120           |
+1.  Open your organization **Security and Risk** page and click the **Configure** button. This opens the configuration page.
+
+1.  In the **Permissions** area, use the search field to find the relevant user and click the user's name.
+
+    ![security and risk management access management](images/security-risk-management-access-management.png)
+
+To revoke the Security Manager role:
+
+1.  Open your organization **Security and Risk** page and click the **Configure** button. This opens the configuration page.
+
+1.  In the **Permissions** area, scroll the list to find the relevant user.
+
+1.  Click the **Revoke role icon** to the right of the user's name and confirm.
+
+<!--
+TODO Update this section once the behavior is consistent. See https://codacy.slack.com/archives/C050XTVETSQ/p1687167901474059
 
 ## Data retention
 
 Codacy retains open items indefinitely and closed items for one year.
 
-Deleting a repository deletes all open items belonging to that repository.
+=== "Codacy"
+
+    Deleting a repository deletes all open items belonging to that repository.
+
+=== "Jira Cloud"
+
+    Uninstalling the Jira Cloud integration deletes all associated open items.
```

Comment:
Uninstalling via Codacy UI will trigger this behavior. But if the integration is uninstalled on Jira directly Codacy will not be notified but no new data will be loaded.